CODE_OF_CONDUCT.md

```diff
@@ -1,5 +1,7 @@
 # Contributor Covenant Code of Conduct
 
+The following is offered as standard code of conduct to which all contributors are assumed to be committed.
```

Comment:
As edits show, the top-level stuff was all inherited from the Metaschema repo and then updated; but I can't recall which updates are due to updated Guidelines and which are editorial decisions.
 
Rather than letting any of these decisions block at this point, I think we should assess whether a review/revision phase of top-level apparatus is called for. Would that be okay @aj-stein-nist?

Reply:
I was just asking why have similar but different stuff we will have to track later, it is minor and non-blocking but worth calling out (to me). I will resolve this one.